packages/rich-text/src/plugins/Table/TableActions.tsx

```diff
@@ -36,26 +38,64 @@ export const TableActions = () => {
   const editor = useContentfulEditor();
   const { onViewportAction } = useTrackingContext();
   const [isOpen, setOpen] = React.useState(false);
+  const [isHeaderEnabled, setHeaderEnabled] = React.useState(false);
 
-  const close = () => {
+  const close = React.useCallback(() => {
     setOpen(false);
 
     if (!editor) return;
 
     // Makes sure we keep the editor in focus when clicking on/out
     // the dropdown menu
     Slate.ReactEditor.focus(editor);
-  };
+  }, [editor]);
+
+  React.useEffect(() => {
+    setHeaderEnabled(editor && isTableHeaderEnabled(editor));
+  }, [editor]);
+
+  const canInsertRowAbove = React.useMemo(() => {
+    if (!editor) {
```

Comment:
We could use `editor.selection` if we want to know if the editor is focused. Sometimes we have `editor` but there's nothing selected.

Reply:
The `getAbove` call should use the `editor.selection` since we don't pass any path. 

But I guess in this particular case it doesn't really matter. We only use the `canInsertRowAbove` to enable/disable items on the actions menu which is only visible if the user focuses on a table cell